dotnet/samples/Demos/TelemetryWithAppInsights/Program.cs

```diff
@@ -94,21 +182,41 @@ private static Kernel GetKernel(ILoggerFactory loggerFactory)
 
         builder.Services.AddSingleton(loggerFactory);
         builder.AddAzureOpenAIChatCompletion(
-            deploymentName: TestConfiguration.AzureOpenAI.ChatDeploymentName,
-            modelId: TestConfiguration.AzureOpenAI.ChatModelId,
-            endpoint: TestConfiguration.AzureOpenAI.Endpoint,
-            apiKey: TestConfiguration.AzureOpenAI.ApiKey
-        ).Build();
+                deploymentName: TestConfiguration.AzureOpenAI.ChatDeploymentName,
+                modelId: TestConfiguration.AzureOpenAI.ChatModelId,
+                endpoint: TestConfiguration.AzureOpenAI.Endpoint,
+                apiKey: TestConfiguration.AzureOpenAI.ApiKey,
+                serviceId: AzureOpenAIChatServiceKey);
+#pragma warning disable SKEXP0070 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
```

Comment:
Can you just add a NoWarn to the csproj for this rather than adding pragmas to the source?